common/lib/xmodule/xmodule/templates/problem/optionresponse_hint.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
 
         Use the following example problem as a model.
 
-        >> A/an ________ is a vegetable.<<
+        >> A/an ________ is a vegetable.||You can select only one option.<<
```

Comment:
Please see the wiki pages! The last two comments on this page, https://openedx.atlassian.net/wiki/display/A11Y/Text+Input provide templates for text input and, separately, for text input with hints and feedback. Please let me know if you notice any problems -- there may be some cut and paste errors(!)
(I believe that replacing these examples with templates is scheduled as a separate task, though.)